surreal/session/default_configs.py

```diff
@@ -179,7 +179,7 @@
     'tensorplex': {
         'host': 'localhost',
         'port': 7008,
-        'tensorboard_port': 6006,
+        'tensorboard_port': 9009,
```

Comment:
Is there a reason we use 9009? @amandlek

Reply:
there was no reason - I just did it when testing to avoid conflict with some other stuff I had on 6006